src/main/resources/lang/simplifiedchinese.lang

```diff
@@ -15,7 +15,7 @@ skript:
 	invalid reload: Skript只能通过Bukkit的“/reload”或Skript的“/skript reload”命令来重新加载。
 	no scripts: 没有发现任何脚本，也许你应该写一些 ;)
 	no errors: 所有脚本都已加载，没有错误。
-	scripts loaded: 已加载%s个脚本，包括%s个结构和%s个命令（%s内）
+	scripts loaded: 已加载 %s 个脚本 总共包含 %s 个结构 (耗时 %s)。
```

Comment:
this doesn't match the provided translation here: https://github.com/SkriptLang/Skript/issues/6928#issuecomment-2239453230

Reply:
```suggestion
	scripts loaded: 已加载 %s 个脚本，总共包含 %s 个结构。 (耗时 %s 秒)
```
As a Chinese myself, improving the grammar of the provided translation

Reply:
Thank you NotSoDelayed!